en/docs/insights/derive-insights.md

```diff
@@ -0,0 +1,364 @@
+
+You can filter the information displayed in the widgets as follows:
+
+| **Filtering Option** | **Description** |
+|--------------------|-------------------------------------------------------------------------------------------------|
+| **By API**         | In the **API** field, you can select the APIs for which you want to view analytics. **All** is selected by default. You can select one or multiple APIs from this selector and view the aggregated result. Once you select an API, you can further filter by a specific application that uses the selected API via the **Application** field described below. |
+| **By Application** | In the **Application** field, you can select the applications for which you want to view analytics. **All** is selected by default. The available applications are all the applications that have subscribed to one or more of the APIs you selected in the **API** field.|
+
+The information you can view is as follows:
+
+ - **API Usage Over Time**
+ 
+    This timeline shows the count of API hits for the APIs you selected using the above-mentioned selectors. If multiple APIs are selected, the timeline shows each API in a separate line with a legend separating each line. You can also zoom in on a selected time range by selecting that area in the chart. To restore to the original view use the **Zoom out** button on the top right-hand corner of the plot.
+
+    ![API usage over time](../assets/img/insights/api-usage-timeline.png){.cInlineImage-full}
+    
+ - **API Usage By Application** 
+ 
+    This widget shows the per-application breakdown of requests for the APIs you selected. You can use the pie chart view or the line chart view. You can switch between the two views using the small icon at the upper-right corner of the widget.
+ 
+    ![API usage by application](../assets/img/insights/usage-by-application.png){.cInlineImage-half}
+ 
+ - **API Usage By Backend**
+ 
+    This widget shows the per-backend breakdown of requests for the APIs you selected. This is useful when multiple APIs are sharing the same backend, which has traffic restrictions. You can use these stats to proactively scale your backends. You can switch between the two views using the small icon at the upper-right corner of the widget.
+ 
+    ![API usage by backend](../assets/img/insights/usage-by-backend.png){.cInlineImage-half}
+ 
+ - **API Resource Usage**
+ 
+    This table shows a resource-level breakdown of API traffic. Each row represents an API resource and it shows the API name, resource path, API method, and the hit count for that combination.
+    
+    ![API resource usage](../assets/img/insights/resource-usage.png){.cInlineImage-half}
+    
+### Errors
+
+This page shows information related to erroneous API calls that are received by your system. The errors are categorization based on the error type. You can further drill down using the error subtypes. Use this page as the starting point for debugging any API errors.
```

Comment:
by Choreo?

Reply:
The errors are categorized?

Reply:
The information in the Insights portal is not just related to API calls received by Choreo, but by many environments including on-premise. Actually, the information in a page at a given time is for a specific combination of organization, environment and tenant. Therefore, used the word "system".